### PR TITLE
[Faucet] Add integrator's guide

### PR DESCRIPTION
**Pull request by banool**
### Description
This PR adds a section to the integrator's guide explaining how to integrate with the faucet as a wallet / SDK dev.

### Test Plan
Docs.

### CODE REVIEW

File: developer-docs-site/docs/guides/system-integrators-guide.md [hunk 1]
```diff
@@ -499,3 +499,47 @@ By monitoring the events, you will find all balance changes in the `0x1::coin::C
 To create some sample data to explore, conduct ["Your first transaction"](../tutorials/first-transaction.md).
 
 To learn more about coin creation, make ["Your First Coin"](../tutorials/first-coin.md).
+
+## Integrating with the faucet
+
+This explains how to integrate with the [Aptos Faucet](https://github.com/aptos-labs/aptos-core/tree/main/crates/aptos-faucet). This is targeted at SDK and wallet creators; if you are a developer using either of these tools, access the faucet through the SDK / wallet instead.
```
Comment (clay-aptos) [inline developer-docs-site/docs/guides/system-integrators-guide.md:505]: I find this reference confusing and am not sure yet how to fix it?
"This is targeted at SDK and wallet creators; if you are a developer using either of these tools, access the faucet through the SDK / wallet instead."

We are saying these instructions are for SDK and wallet folks and then if you are using those tools to do something else instead?

What is the instead for? Sorry if I am being dense.

**Review by clay-aptos — APPROVED** [review]
Thanks, Daniel. This LGTM with some small changes I made directly and one question I have as a comment. I am adding David and Greg as I recall David wanting Eng review on docs changes. Thanks again!

File: developer-docs-site/docs/guides/system-integrators-guide.md (hunk 1 above)
Comment (saharct) [reply developer-docs-site/docs/guides/system-integrators-guide.md:505]: I have the same question, @banool. Please clarify if you meant something like "This tutorial explains how SDK and wallet creators can integrate with the Aptos Faucet. If you are a dapp developer, access the faucet through an existing [SDK](https://aptos.dev/tutorials/your-first-transaction/) or [wallet](https://aptos.dev/guides/get-test-funds) instead."

Comment (banool) [reply developer-docs-site/docs/guides/system-integrators-guide.md:505]: Feel free to make whatever changes you think make this clearer and I can take a look!

Comment (davidiw) [reply developer-docs-site/docs/guides/system-integrators-guide.md:505]: Please let's address this before landing.

Comment (saharct) [reply developer-docs-site/docs/guides/system-integrators-guide.md]: @banool I rewrote that portion. Please review it and let me know if any changes are needed. Thanks!

Comment (banool) [reply developer-docs-site/docs/guides/system-integrators-guide.md]: Looks great, thanks a lot!

**Review by saharct — APPROVED** [review]

**Comment by clay-aptos** [comment]
Just noting I am announcing this section and adding it to the **Interact with the Aptos Blockchain* index along with other updates in:
https://github.com/aptos-labs/aptos-core/pull/7415

Thanks, @banool !

**Pull request merged**